src/Internal/UriValidation/pch.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+
+#define NOMINMAX

```

Comment:
This seems like a copy/paste that doesn't need to have so much in it.  Given the single file nature of this lib, it doesn't even need a precompiled header.